maker-core/delegates.md

```diff
@@ -1,5 +1,39 @@
-# TBC
+# Constitutional Delegates (CDs)
 
-{% hint style="info" %}
-Sorry! We haven't been able to publish documentation on this topic yet. Please check back soon.
-{% endhint %}
+## Overview
+Delegates are ecosystem actors who operate through Protocol Delegation System smart contracts. MKR holders can delegate their voting power to them without the need to hand over the custody of their MKR.
+
+Constitutional Delegates (CDs) are one type of [Constitutional Conservers](https://endgame.makerdao.com/maker-core/tbc-governance/constitutional-conservers). They receive various benefits and are subject to specific requirements. They must put in place the governance strategies defined by the CVCs.
+
+CDs must not engage with other ecosystem actors or in operational activities. They are not allowed to compensate MKR holders that delegate to them. They must aim to provide governance information and research material to MKR holders.
+
+## Purpose
+
+The Constitutional Delegate role spares Delegates from having to create governance strategies. CDs must follow strategies that have been designed by CVC members.
+CDs do not take part in the creation of the strategies, they execute them. The risk of Delegates following their own agenda, since they are not required to hold any MKR, is then  eliminated.
```

Comment:
Not sure if this comes under "Purpose". I'd say something along the lines of "CDs exist in order to allow MKR holders to delegate their MKR and also support a CVC strategy." or something